log.go

```diff
@@ -14,22 +14,25 @@ var (
 	spawn      = panics.GoroutineWrapperFunc(log)
 )
 
-func initLog(logFile, errLogFile string) {
-	err := backendLog.AddLogFile(logFile, logger.LevelTrace)
+func initLog(noLogFiles bool, logLevel, logFile, errLogFile string) {
+	level, _ := logger.LevelFromString(logLevel)
```

Comment:
In case of parsing failure, it should be better to display a message to the console and stop the application. Otherwise, you fallback to info level without being aware of the fact, which can be misleading.

Reply:
Fixed.